docker.Makefile

```diff
@@ -37,7 +37,8 @@ cross: build_docker_image
 
 # start container in interactive mode for in-container development
 dev: build_docker_image
-	@docker run -ti $(MOUNTS)  $(DEV_DOCKER_IMAGE_NAME) ash
+	@docker run -ti $(MOUNTS) -v /var/run/docker.sock:/var/run/docker.sock $(DEV_DOCKER_IMAGE_NAME) ash
```

Comment:
I guess we want this for `cross` as well, otherwise we won't get the binaries.

Want to add that now, or later?